doc/test-framework/writing-e2e-tests.md

```diff
@@ -275,6 +275,12 @@ $ kubectl create -f deploy/operator.yaml --namespace operator-test
 $ operator-sdk test local ./test/e2e --namespace operator-test --no-setup
 ```
 
+### Test user setup
+
+Note that the Service Account the test is run under must have sufficient permissions to create and use the resources needed in the test. The simplest way to
+accomplish this is to bind the cluster-admin Cluster Role to the Service Account you will run the test under. If you are unable or unwilling to grant such access,
+a more limited Cluster Role such as this [`testuser_clusterrole.yaml`][testuser-clusterrole] can be created and bound to the Service Account you are using.
```

Comment:
Instead of creating an additional file, I think would be better it has the example in the doc. 

```suggestion
a more limited Cluster Role such as : 

```yaml
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRole
metadata:
  name: testuser
rules:
- apiGroups:
  - ""
  - apiextensions.k8s.io
  - cache.example.com # the api space your tests are created in
  - apps
  resources:
  - memcacheds #the type of the CRD in your operator
  - namespaces
  - customresourcedefinitions
  - deployments
  - pods
  verbs:
  - create
  - delete
  - get
  - list
  - watch
  - update

```

Also, why are you suggesting to create a cluster role a not a Role?

Reply:
Ok, I can inline it. As for why a ClusterRole, the default framework tests spin up new namespaces to run the tests in. So you'd have to have ClusterRole access to create those.

Reply:
It is true. Good point. It will create namespaces. 
Would be nice clarifies it as well. I mean, why a cluster role is required and the mandatory permissions in this case.

Then, I understand that we are asking about apply the operator permissions to the user which will run the tests. Am I right? So, I think that the example is not required at all. We can just say that. WDYT? 

For example, for we do the same for the [Memcached Go Sample](https://github.com/operator-framework/operator-sdk-samples/blob/master/go/memcached-operator/). The role for the user should have the same permissions of the [deploy/role](https://github.com/operator-framework/operator-sdk-samples/blob/master/go/memcached-operator/deploy/role.yaml) applied for the operator.

Reply:
I determined through manual testing what the minimum permissions needed were. If we wanted to keep things simple, I could have just said they needed to add cluster-admin permissions, but in the linked issue it sounded like we wanted to find what the minimum required were.

Reply:
I understand that the permissions required are the same operator's permissions. And the, only additional mandatory permission is the namespace to allow it to create new ones. Otherwise, it will not works.

Reply:
something like this?